rules-reviewed/hibernate/hibernate.windup.xml

```diff
@@ -1,68 +1,163 @@
 <?xml version="1.0"?>
 <ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hibernate" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+   xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
 
-    <metadata>
-        <description>
+   <metadata>
+      <description>
             This ruleset provides analysis of deprecated Hibernate java constructs and their migration to newer one.
         </description>
-        <dependencies>
-            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.3.0.Final" />
-            <addon id="org.jboss.windup.rules,windup-rules-java,2.3.0.Final" />
-        </dependencies>
-        <sourceTechnology id="hibernate" versionRange="[,3.9)" />
-        <targetTechnology id="hibernate" versionRange="[4,)" />
-        <tag>reviewed-2015-06-18</tag>
-    </metadata>
+      <dependencies>
+         <addon id="org.jboss.windup.rules,windup-rules-javaee,2.3.0.Final" />
+         <addon id="org.jboss.windup.rules,windup-rules-java,2.3.0.Final" />
+      </dependencies>
+      <sourceTechnology id="hibernate" versionRange="[,3.9)" />
+      <targetTechnology id="hibernate" versionRange="[4,)" />
```

Comment:
I think adding a line for EAP would be good. It can have more than one target technology.